lib/site_prism/waiter.rb

```diff
@@ -14,5 +14,12 @@ def self.wait_until_true(wait_time = Capybara.default_max_wait_time)
 
       raise SitePrism::TimeoutError, "Timed out after #{wait_time}s."
     end
+
+    def self.wait_until_displayed?(*args, url, &url_matches)
```

Comment:
This won't be a predicate, so lets drop the `?`

Reply:
Removed '?' from the method